mesmer/stats/_auto_regression.py

```diff
@@ -81,26 +83,46 @@ def _fit_auto_regression_scen_ens(*objs, dim, ens_dim, lags):
 
     Notes
     -----
-    Calculates the mean auto regression, first over the ensemble members, then over all
-    scenarios.
+    If `ens_dim` is not `None`, calculates the mean auto regression first over all ensemble
+    members and then over scenarios. This is done to weight scenarios equally, consequently
+    ensemble members are not weighted equally, if the number of members differs between scenarios.
+    If no ensemble members are provided, the mean is calculated over scenarios only.
     """
 
+    def _avg_ar_params(ar_params, dim, nobs):
+        ar_params["coeffs"] = ar_params.coeffs.mean(dim)
+        ar_params["intercept"] = ar_params.intercept.mean(dim)
+        ar_params["variance"] = (ar_params.variance * nobs).sum(dim=dim) / nobs.sum(
+            dim=dim
+        )
+        return ar_params
+
     ar_params_scen = list()
-    for obj in objs:
-        ar_params = fit_auto_regression(obj, dim=dim, lags=int(lags))
+    n_ens = np.ones(len(objs))
 
-        # BUG/ TODO: fix for v1, see https://github.com/MESMER-group/mesmer/issues/307
-        ar_params["standard_deviation"] = np.sqrt(ar_params.variance)
+    for o, obj in enumerate(objs):
```

Comment:
```suggestion
    for i, obj in enumerate(objs):
```

(less confusing than `o`)